docker/README.md

```diff
@@ -258,6 +260,30 @@ You can now start both tunnels:
 ngrok start jetpack jetpack-sftp
 ```
 
+### Docker ngrok
```

Comment:
We could switch "Configuration file" and "Docker ngrok" other way around so that Docker version is the primary/default way to go and if folks need more granular control, they could use the config file?

And a nitpick: Ngrok with a capital N would be ace. :-)

Reply:
Yes, I would love so to, but I didn't want to be the one suggesting it :D

Reply:
Done!